contrib/serializers/doctorate.py

```diff
@@ -1,13 +1,15 @@
+from django.utils.translation import gettext_lazy as _
 from rest_framework import serializers
 
-from admission.contrib.models import AdmissionDoctorate
+from admission.contrib.models import AdmissionDoctorate, AdmissionType
+from base.models.person import Person
 
 
-class AdmissionDoctorateSerializer(serializers.ModelSerializer):
-    url = serializers.URLField(source="get_absolute_url")
-    type = serializers.CharField(source="get_type_display")
-    candidate = serializers.StringRelatedField()
-    author = serializers.StringRelatedField()
+class AdmissionDoctorateReadSerializer(serializers.ModelSerializer):
+    url = serializers.URLField(source="get_absolute_url", read_only=True)
+    type = serializers.CharField(source="get_type_display", read_only=True)
+    candidate = serializers.StringRelatedField(read_only=True)
+    author = serializers.StringRelatedField(read_only=True)
```

Comment:
Les StringRelatedField sont read only par défaut

```suggestion
    url = serializers.ReadOnlyField(source="get_absolute_url")
    type = serializers.ReadOnlyField(source="get_type_display")
    candidate = serializers.StringRelatedField()
    author = serializers.StringRelatedField()
```